test/test_complex.py

```diff
@@ -30,10 +30,8 @@ def test_copy_real_imag_methods(self):
         real = torch.randn(4)
         imag = torch.randn(4)
         complex_tensor = real + 1j * imag
-        # TODO(#38095): Replace assertEqualIgnoreType. See issue #38095
-        self.assertEqualIgnoreType(complex_tensor.copy_real(), real)
-        # TODO(#38095): Replace assertEqualIgnoreType. See issue #38095
-        self.assertEqualIgnoreType(complex_tensor.copy_imag(), imag)
+        self.assertEqual(complex_tensor.copy_real(), real + 1j * 0)
+        self.assertEqual(complex_tensor.copy_imag(), 0 + 1j * imag)
```

Comment:
`self.assertEqual(complex_tensor.copy_imag(), imag + 0j)`

`.copy_real()` and `.copy_imag()` return a new complex tensor with imaginary component equal to zero and real component equal to the real and imaginary values of the complex_tensor. These methods will ultimately be removed when storage becomes untyped and `real` and `imag` would then be made tensor properties which would perform a view.

Reply:
In this case (since tests are failing) I assume we have something wrong with this functions. I can take a look later.

Reply:
you need to compare `complex_tensor.copy_imag()` with `imag + 0j` not `0 + 1j * imag`

Reply:
alternately you could also do:
```
self.assertEqual(complex_tensor.copy_real().to(torch.float), real)
self.assertEqual(complex_tensor.copy_imag().to(torch.float, imag)
```